functions.js

```diff
@@ -0,0 +1,38 @@
+let addIngredient = () => {
```

Comment:
I'd recommend naming this file something a bit more descriptive like recipe.js or jsonConverter.js. When this is your only js file it's not too bad, but when we get to React the number of files will multiply exponentially so I'd recommend getting in the habit of thinking of good names for your file now.